src/main/sensors/gyro.h

```diff
@@ -60,10 +62,10 @@ typedef union gyroLowpassFilter_u {
 
 typedef enum gyroDetectionFlags_e {
     GYRO_NONE_MASK = 0,
-    GYRO_1_MASK = BIT(0),
-#if defined(USE_MULTI_GYRO)
-    GYRO_2_MASK = BIT(1),
-    GYRO_ALL_MASK = (GYRO_1_MASK | GYRO_2_MASK),
+#if GYRO_COUNT == 2
+    GYRO_IDENTICAL_MASK = BIT(7), // All gyros are of the same hardware type
```

Comment:
GYRO_ALL_MASK = (1 << GYRO_COUNT) - 1;

Reply:
Ahh yes that is certainly better. Thanks for the find.